From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Wiz <95232096+GameCubeGBA@users.noreply.github.com>
Date: Sat, 11 Dec 2021 12:55:37 -0500
Subject: [PATCH] 3745: Insert placeholder for FE DE FF E7 opcode

This is for issue #3723, since this appears to be a debugger ptrace breakpoint.
---
 .../AARCH64/data/languages/AARCH64instructions.sinc        | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Ghidra/Processors/AARCH64/data/languages/AARCH64instructions.sinc b/Ghidra/Processors/AARCH64/data/languages/AARCH64instructions.sinc
index 2f62006d8..cde9f7092 100644
--- a/Ghidra/Processors/AARCH64/data/languages/AARCH64instructions.sinc
+++ b/Ghidra/Processors/AARCH64/data/languages/AARCH64instructions.sinc
@@ -3999,5 +3999,10 @@ unimpl
 is b_0031=0x00200c20
 unimpl
 
-} # end with ImmS_ImmR_TestSet=1
+# TODO This is a placeholder until the correction instruction implementations can be found
+:und_bkpt
+is b_0031=0xE7FFDEFE
+{
+}
 
+} # end with ImmS_ImmR_TestSet=1
-- 
2.38.1

